apps/site/assets/ts/shuttles/__tests__/diversionData.json

```diff
@@ -0,0 +1 @@
37101,-71.11422],[42.372409999999995,-71.11581],[42.372479999999996,-71.1159],[42.37251,-71.11594],[42.37327,-71.11847999999999],[42.373329999999996,-71.11863999999998],[42.37338999999999,-71.11875999999998],[42.373459999999994,-71.11881999999999],[42.373509999999996,-71.11883999999999],[42.37367,-71.11883999999999],[42.374039999999994,-71.11876],[42.374039999999994,-71.11876],[42.37441999999999,-71.11868],[42.37496999999999,-71.11852999999999],[42.37548999999999,-71.11849],[42.375719999999994,-71.11847999999999],[42.375859999999996,-71.11852999999999],[42.37596,-71.11859999999999],[42.3761,-71.11875999999998],[42.37627,-71.11906999999998],[42.37646,-71.11942999999998],[42.3767,-71.11968999999998],[42.37723,-71.11989999999997],[42.37772,-71.12005999999997],[42.378029999999995,-71.12007999999997],[42.378409999999995,-71.12003999999997],[42.38081999999999,-71.11978999999998],[42.388189999999994,-71.11893999999998],[42.388279999999995,-71.11893999999998],[42.388279999999995,-71.11893999999998],[42.39061999999999,-71.11882999999997],[42.393629999999995,-71.11914999999998],[42.39437999999999,-71.11920999999998],[42.39495999999999,-71.11923999999998],[42.39519999999999,-71.11932999999998],[42.39537999999999,-71.11945999999998],[42.39560999999999,-71.11968999999998],[42.39583999999999,-71.12002999999997],[42.39616999999999,-71.12062999999998],[42.39643999999999,-71.12119999999997],[42.39658999999999,-71.12156999999998],[42.396669999999986,-71.12185999999998],[42.396779999999985,-71.12220999999998],[42.39683999999998,-71.12246999999998],[42.39683999999998,-71.12246999999998],[42.397049999999986,-71.12329999999997],[42.39720999999999,-71.12416999999998],[42.397269999999985,-71.12471999999998],[42.39734999999998,-71.12544999999999],[42.39743999999998,-71.12646999999998],[42.397539999999985,-71.12802999999998],[42.39759999999998,-71.12909999999998],[42.39772999999998,-71.13087999999998],[42.39792999999998,-71.13449999999997],[42.39796999999998,-71.13519999999997],[42.397869999999976,-71.13607999999996],[42.39782999999998,-71.13635999999997],[42.39774999999998,-71.13669999999996],[42.39765999999998,-71.13707999999997],[42.39760999999998,-71.13717999999997],[42.39755999999998,-71.13724999999997],[42.397489999999976,-71.13728999999996],[42.397219999999976,-71.13736999999996],[42.396949999999975,-71.13748999999996],[42.396679999999975,-71.13770999999996],[42.396479999999976,-71.13801999999995],[42.39635999999997,-71.13828999999996],[42.39634999999997,-71.13831999999995],[42.39624999999997,-71.13895999999995],[42.39623999999996,-71.13913999999995],[42.39618999999996,-71.13954999999996],[42.39614999999996,-71.14207999999995]],"is_shuttle_route":false,"id":"931_0010-70082-70072-2","direction_id":1},{"polyline":[[42.39615,-71.14208],[42.39619,-71.13955000000001],[42.39624,-71.13914000000001],[42.39625,-71.13896000000001],[42.396350000000005,-71.13832000000001],[42.39636000000001,-71.13829000000001],[42.39648000000001,-71.13802000000001],[42.39668000000001,-71.13771
```

Comment:
We should refresh this snapshot, if we haven't already, so it correctly excludes the extra shapes that were in there before.

Reply:
Good call, will update this fixture along with the associated snapshots!